jspm.config.js

```diff
@@ -1,4 +1,10 @@
 SystemJS.config({
+  browserConfig: {
```

Comment:
Hm... yes, you did that. Haven't seen that immediately. I've updated the commit to call `npm run touch-jspm-browser` within the `dev-bundle` task, before jspm bundle is started. Removed the `browserConfig` entry as well.